src/cli/subcmds/deploy.rs

```diff
@@ -118,21 +118,25 @@ fn zip_dir(dir: &str, target_file: &str) -> Result<(), Error> {
     Ok(())
 }
 
-/// Compresses and deploys the project as a distributable program.
-pub fn execute(matches: &ArgMatches) -> cargo::CmdResult {
-    try!(::subcmds::test::execute(matches));
-    match ::subcmds::build::execute(matches) {
-        Ok(a) => {
-            tryio!(setup_deploy_dir());
-
-            // Compress Resources to zipfile in deploy directory
-            tryio!(zip_dir(RESOURCES_DIR, &Path::new(DEPLOY_DIR).join(RESOURCES_ZIP_FILENAME).to_str().unwrap()));
-
-            // Copy compiled binaries - Amethyst system dynamic libraries and executable
-            tryio!(copy_binaries(&Path::new(BUILD_DIR).to_str().unwrap(), &Path::new(DEPLOY_DIR).to_str().unwrap()));
-
-            Ok(a)
-        },
-        Err(e) => Err(e),
+pub struct Cmd;
+impl AmethystCmd for Cmd {
+    /// Compresses and deploys the project as a distributable program.
+    fn execute<I: AmethystArgs>(matches: &I) -> cargo::CmdResult {
+        let cargo_args = vec!["--release"];
```

Comment:
Once again, build and test subcommands look for `release` argument, not `--release`.